scripts/get-quay-token

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo -n "Username: "
+read USERNAME
```

Comment:
Please take this input as command line parameters. You should also add a help option.

Reply:
This can be done in a follow up PR